packages/shared/lib/services/connection.service.ts

```diff
@@ -1451,6 +1544,67 @@ class ConnectionService {
         }
     }
 
+    public getJwtCredentials(
+        provider: ProviderJwt,
+        privateKeyId?: string,
+        privateKey?: string,
+        issuerId?: string,
+        apiKey?: string
+    ): ServiceResponse<JWTCredentials> {
+        if (apiKey) {
+            const apiKeyParts = apiKey.split(':');
+            if (apiKeyParts.length === 2 && apiKeyParts[0] && apiKeyParts[1]) {
+                privateKeyId = apiKeyParts[0];
+                privateKey = apiKeyParts[1];
+            } else {
+                throw new NangoError(`invalid_api_key_format`);
+            }
+        }
```

Comment:
you can do 
```
if (apiKey) {
  const [privateKeyId, privateKey] = apiKey.split(':');
  
  if (!privateKeyId || !privateKey) {
    throw new NangoError('invalid_api_key_format');
  }
}
```

Reply:
we should also document that when specified we expect this attribute to be the id/secret separated by a colon character

Reply:
Yes, we should have this documented in the providers docs, but now that `apiKey` is not part of `JwtCredentials`, I have updated the logic to split the `privateKey` at the webapp, that way we can have 

```
interface JWTCredentials {
    type: AuthModes['Jwt'];
    privateKey: {
       id: string,
       secret: string
    } | string, // colon separated string: 'id:secret'
    issuerId?: string;
    token?: string;
    expires_at?: Date | undefined;